components/movementsensor/rtkutils/cachedData.go

```diff
@@ -217,7 +217,9 @@ func (g *CachedData) calculateCompassDegreeError() float64 {
 	if adjacent == 0 {
 		return math.NaN()
 	}
+	// by default we assume fix is 1-3. In this case, we assume radius to be 5m.
 	radius := 5.0
+	// when fix is 4 or higher, we set radius to be 10cm.
```

Comment:
The comments say what we're doing, but I could have gotten that just as easily from the code. I still don't understand _why_ we're doing any of this. What circle does this radius go with? Why are 5m and 10cm the right lengths for it?

Ideally, comments are like a director's commentary on a film, explaining the choices behind what's in the film/code, not simply what's in the film/code (which you can get yourself from watching/reading). These comments add nothing beyond the code itself, but if they explained _why_ the code should do these things, it would be much more useful!